README.md

```diff
@@ -30,12 +30,37 @@ jobs:
     - uses: actions/checkout@v2
     - name: Setup tmate session
       uses: mxschmitt/action-tmate@v3
+      with:
+        limit-access-to-actor: true
 ```
 
 To get the connection string, just open the `Checks` tab in your Pull Request and scroll to the bottom. There you can connect either directly per SSH or via a web based terminal.
 
 ![GitHub Checks tab](./docs/checks-tab.png "GitHub Checks tab")
 
+## Security consideration: Use registered public SSH key(s)
+
+By default *anybody* can connect to the tmate session.
+This can lead to security issues, for example if secrets are available in environment variables or in subsequent steps privileged code is executed.
```

Comment:
Here, we need to inform users that secrets are _not_ made available by default, but that the risk comes from configuring `token` in the `action-tmate` Action or from adding `${{ secret.X }}` values to the `env` block, or from writing out secret values to disk in preceding workflow steps.

Reply:
for potential follow-uppers:
... or from configurations that are generated/downloaded/... by previous steps having access to secrets. Or from subsequent steps deploying signed code that has been produced including injected bits or from a whole variety of other reasons we did not think about right now, and probably the 90% of users who don't understand SSH keys will neither understand so we better stay on the safe side and try to be careful if in doubt.